.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,14 +29,39 @@ Short description explaining the high-level reason for the pull request
 
 ## Checklist
 
-* [ ] PR has an informative and human-readable title
-* [ ] Changes are limited to a single goal (no scope creep)
-* [ ] Code can be automatically merged (no conflicts)
-* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
-* [ ] Passes all existing automated tests
-* [ ] Any *change* in functionality is tested
-* [ ] New functions are documented (with a description, list of inputs, and expected output)
-* [ ] Placeholder code is flagged
-* [ ] Visually tested in supported browsers and devices
-* [ ] Project documentation has been updated
-* [ ] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:
+- [ ] PR has an informative and human-readable title
+- [ ] Changes are limited to a single goal (no scope creep)
+- [ ] Code can be automatically merged (no conflicts)
+- [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
+- [ ] Passes all existing automated tests
+- [ ] Any _change_ in functionality is tested
+- [ ] New functions are documented (with a description, list of inputs, and expected output)
+- [ ] Placeholder code is flagged / future todos are captured in comments
+- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
+- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
```

Comment:
Remove the parenthetical on this one for this project.

Reply:
Ah right!

Reply:
Fixed